seed/static/seed/js/services/matching_service.js

```diff
@@ -24,6 +24,8 @@ angular.module('BE.seed.service.matching', []).factory('matching_service', [
         params: { organization_id: user_service.get_organization().id }
       }).then(function (response) {
         return response.data;
+      }).catch(function(e) {
+        if (e.status === 403) return e.data;
```

Comment:
Did you encounter an uncaught exception that made you make this change? I think your change is likely fine as it is, but am just hoping to get more background here.

Reply:
I did. And i recall this is after/skip importing file step and when a user attempts to manually geocode selected properties/taxlots on inventory list, error message wasn't caught and prompted prior to this change. So in situation where a user changes the api key after importing a file and tried to manually geocode, an error could be missed.

Reply:
@adrian-lara this seems a bit of a tricky situation. Assuming a user imported a file using a valid key, between he/she manually geocodes selected properties/tax lots on inventory list he/she accidentally changed the api key to an invalid one. In this case, geocoding attempts have indeed been made (from successfully importing a file prior) but will fail this time when he/she hits "Geocode" button to geocode again. But this time, error message pops up.
I could have been over-thinking again I guess this comes down to how possible it is for users to alter their mapquest api keys.

Reply:
> I could be over-thinking here butI wonder how possible it is for users to alter their keys in between confirm and saving their mapping and manually geocode on inventory lists?

I think I'm missing something, but this scenario shouldn't present a problem? If a user ever has an invalid key, they should get the appropriate message whenever trying to geocode records using their address fields and MapQuest.